server/src/test/java/org/elasticsearch/common/rounding/RoundingDuelTests.java

```diff
@@ -42,6 +43,7 @@ public void testSerialization() throws Exception {
             rounding = org.elasticsearch.common.Rounding.builder(timeValue()).timeZone(ZoneOffset.UTC).build();
         }
         BytesStreamOutput output = new BytesStreamOutput();
+        output.setVersion(Version.V_6_4_0);
```

Comment:
I guess this is to force the old behavior. Does it make sense to add a comment here why this version has been chosen?

Reply:
using `VersionUtils.getPreviousVersion(Version.V_7_0_0)` now